ndmg/register/gen_reg.py

```diff
@@ -960,297 +967,3 @@ def dwi2atlas(self, clean=False):
             )
             print("Cleaning temporary registration files...")
             os.system(cmd)
-
-
-class epi_register(object):
```

Comment:
big deletion, but sounds good.
It's possible that we can PR these big functions into something so that they don't go to waste, but honestly with the fMRI stuff might be better to just let it burn

Reply:
But are we going to ever actually use them? fMRI is going to be all cpac, so all this registration is going to be handled by that. Nothing happening in these functions is all that groundbreaking and worth keeping, in my opinion